README.md

```diff
@@ -115,6 +115,28 @@ Official [Buildkite Test Analytics](https://buildkite.com/test-analytics) collec
      }
    ```
 
+   ### Cypress
+
+   Update your [Cypress configuration](https://docs.cypress.io/guides/references/configuration):<br>
+
+   ```js
+    // cypress.config.js
+
+    // Send results to Test Analytics
+   reporter: "test-collector-javascript/cypress/reporter,
+   ```
+
+   If you would like to pass in the API token using a custom environment variable, you can do so using the reporterOptions.
+
+   ```js
+   // cypress.config.js
+
+   // Send results to Test Analytics
+   reporterOptions: {
+    token_name: "CUSTOM_ENV_VAR_NAME"
```

Comment:
What do you think of passing the `token` directly to the option instead of env name, similar to what `jest/reporter` does?
```suggestion
    token_name: process.env.BUILDKITE_ANALYTICS_TOKEN
```